### PR TITLE
Updating license ISC to GPL3 in ./backend

### PR DESCRIPTION
**Pull request by vSt11**
Like specified, it needed to be simply updated

### CODE REVIEW

**Review by ctmbl — CHANGES_REQUESTED** [review]

**Comment by ctmbl** [comment]
I forgot to mention it, but maybe we should also update `author`? see [the doc](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors) about `person` `author` and `contributors`, but I'm fine with letting `Alexandre Tullot` for now and further updating it on later PRs (contributing to the backend I mean)

File: backend/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "version": "npm install && git add package.json package-lock.json"
   },
   "author": "Alexandre Tullot",
-  "license": "ISC",
+  "license": "LICENSE.MD",
```
[the remark this answers is not in the document]
Comment (amtoine) [reply backend/package.json:13]: :laughing: 
then it's the type of license and not the license filename :+1:

**Comment by vSt11** [comment]
For the author field, do you allow me to give your email @atxr ? If so, you can reply it to me here then. 
I don't really know all the people I should credit for now, maybe it's better to update it later, in a different issue ?

**Review by ctmbl — APPROVED** [review]
After reading [our license](https://github.com/iScsc/iscsc.fr/blob/main/LICENSE.MD) (really just the beginning) I think sticking with `GPL-3.0-only` is fine :+1: 

As for the `author` field I would like to see @atxr opinion first :wink:
But as is this PR is good to merge according to me!

**Review by atxr — COMMENTED** [review]
Aha I'm totally fine about being the author of this repo, even if it's the whole club :wink: 
About the mail, I don't have any problem, but I think putting the iscsc mail would be better
supaerocsclub@gmail.com

**Review by atxr — APPROVED** [review]
@ctmbl is right, this author discussion is out of the scope of #50 
Let's merge this :rocket:

**Pull request merged**